src/99-appendices/11-qmri.md

```diff
@@ -232,6 +232,15 @@ For example, without the information of `MagneticFieldStrength`, white-matter T1
 -   The JSON file accompanying a qMRI map which is obtained by using open-source software is RECOMMENDED
     to include additional metadata fields listed in the following table:
 
+{{ MACROS___make_metadata_table(
+   {
+      "BasedOn": "RECOMMENDED",
+      "EstimationReference": "RECOMMENDED",
+      "EstimationAlgorithm": "RECOMMENDED",
+      "Units": "RECOMMENDED",
```

Comment:
these look good to me in principle, but a lot of content has been added/edited (req level, datatype, description), so we should double check these with Agah et al. :-)

Reply:
That's a good point. @agahkarakuzu would you mind taking a quick look at the new (auto-generated) table to see if it looks good to you? I just attempted to render the same information, but in a format that matches the way the rest of the specification represents metadata information.

## Old, hardcoded table

![image](https://user-images.githubusercontent.com/8228902/123830000-eadbc280-d8d0-11eb-8eca-a6a4b1e0a102.png)

## New, auto-generated table

![image](https://user-images.githubusercontent.com/8228902/123830042-f202d080-d8d0-11eb-95f9-f7862df0f748.png)

Reply:
@tsalo these look great, no issues with the Data Type or Description.

Reply:
Thanks @agahkarakuzu! I will resolve this thread then.